pkg/solana/chainwriter/chain_writer.go

```diff
@@ -282,12 +274,17 @@ func (s *SolanaChainWriterService) SubmitTransaction(ctx context.Context, contra
 		return errorWithDebugID(fmt.Errorf("error parsing fee payer address: %w", err), debugID)
 	}
 
-	accounts = append([]*solana.AccountMeta{solana.Meta(feePayer).SIGNER().WRITE()}, accounts...)
-	accounts = append(accounts, solana.Meta(solana.SystemProgramID))
```

Comment:
Just wanted to confirm that we didn't need to add these to accounts manually anymore

Reply:
No - I made the mistake originally that the feePayer is always the first account and the system program is always last. That's not necessarily true